outputs.tf

```
@@ -0,0 +1,4 @@
+output "management_locks" {
+  description = "the management locks created by this module"
+  value       = azurerm_management_lock.protected_resource_lock
+}
```

It's not recommended to return resources directly.  Consider using ID:
```suggestion
output "management_lock_ids" {
  description = "ids of the the management locks created by this module"
  value       = [for lock in azurerm_management_lock.protected_resource_lock : lock.id ]
}
```